content/en/docs/languages/java/api.md

```diff
@@ -610,67 +610,136 @@ public class OpenTelemetryUsage {
 
 ### GlobalOpenTelemetry
 
-{{% alert title="Java agent" %}} The Java agent is a special case where
```

Comment:
Can we add a replacement tl;dr for the impacient?

Reply:
Gotcha. I think my guidance to these users would still to be use `GlobalOpenTelemetry.getOrNoop()`.